### DISCUSSION TITLE
`class BaseClass:` or `class BaseClass(object)`

### DISCUSSION THREAD

**Discussion by cidrblock** [ansible/ansible-navigator#700] [Linters, linting, formatting, pre-commit, CI]
```
lint-vetting: WPS306 Found class without a base class: BaseClass
```

conflicts with:

```
R0205: Class 'BaseClass' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
```

So either we disable WPS306 _or_ the pylint R0205.

My preference would be the WPS error because, if only because it feels like a step backward, even knowing the message is not related to python2 but instead consistency.

**Comment by webknjaz** [comment]
FWIW never inheriting from `object` is also consistent. I'd like to disable WPS306 with a comment that we chose a different side of this consistency.

**Reply by ssbarnea** [reply]
Yep, the future is omitting implicit object inheritance. We already went this way for the other projects. 

The only thing prevented that was py2, which is no longer a problem.

Ok to add the related WPS option to deliberately disabled section.

**Reply by webknjaz** [reply]
> The only thing prevented that was py2, which is no longer a problem.

Technically, py2 doesn't prevent this, it just needs `__metaclass__ = type` boilerplate in the modules.
